### PR TITLE
Now stores skills as enums, can process multiple characters

### PR DESCRIPTION
**Pull request by maliceholmes**

### CODE REVIEW

File: Character.py [hunk 1]
```diff
@@ -1,18 +1,70 @@
...
+            character.Proficiencies = []
+            for p in list(k['isProficient']):
+                character.Proficiencies.append(getEnum(p))
+            character.Expertise = []
+            for e in list(k['isExpert']):
+                character.Expertise.append(getEnum(e))
+            characterList.append(character)
+        return characterList
+
+def getEnum(name):
```
Comment (kpwelsh) [inline Character.py:46]: See Enum documentation
https://docs.python.org/3/library/enum.html#programmatic-access-to-enumeration-members-and-their-attributes

**Review by kpwelsh — CHANGES_REQUESTED** [review]

File: Character.py (hunk 1 above)
Comment (maliceholmes) [reply Character.py:46]: fixed

Comment (kpwelsh) [reply Character.py:46]: can you get rid of this function then?

**Comment by kpwelsh** [comment]
I put this in when making other changes.

**Pull request closed without merging**